backend/LexBoxApi/GraphQL/CustomTypes/RefreshJwtOrgMembershipMiddleware.cs

```diff
@@ -0,0 +1,77 @@
+using HotChocolate.Resolvers;
+using LexBoxApi.Auth;
+using LexCore.Auth;
+using LexCore.Entities;
+
+namespace LexBoxApi.GraphQL.CustomTypes;
+
+public class RefreshJwtOrgMembershipMiddleware(FieldDelegate next)
+{
+    private const string REFRESHED_USER_KEY = "RefreshedJwtMemberships";
+    // Should be the same value as in RefreshJwtProjectMembershipMiddleware.
```

Comment:
Would like to move this to a static public const on some class that both RefreshJwtProjectMembershipMiddleware and RefreshJwtOrgMembershipMiddleware will reference. @hahn-kev, I'd welcome your suggestions as to where that class should live, name, etc.

Reply:
yeah that makes sense, put it in `GraphQlSetupKernel`

Reply:
@rmunn this seems to have been forgotten

Reply:
Not forgotten, just didn't spot https://github.com/sillsdev/languageforge-lexbox/pull/888#discussion_r1663380935 because it didn't show up in the review comments lower down. Fixed now in commit 3909e11d.

Reply:
Whoops, 3909e11 was based off of the wrong branch. Actual commit fixing this is 79859a9f40d173276e5526be14b139ba04a7149e.